docs/en/sql-reference/sql-functions/dict-functions/dict_mapping.md

```diff
@@ -0,0 +1,215 @@
+---
+displayed_sidebar: "English"
+---
+
+# dict_mapping
+
+## Description
+
+Returns the value mapped to the specified key in a dictionary table.
+
+This function is mainly used to simplify the application of a global dictionary table. During data loading into a target table, StarRocks automatically obtains the value mapped to the specified key from the dictionary table by using the input parameters in this function, and then loads the value into the target table.
+
+Note that currently StarRocks's shared-data mode does not support this function.
+
+## Syntax
+
+```SQL
+dict_mapping("[<db_name>.]<dict_table>", key_column_expr_list [, <value_column> ] [, <strict_mode>] )
+
+key_column_expr_list ::= key_column_expr [, key_column_expr ... ]
+
+key_column_expr ::= <column_name> | <expr>
+```
+
+## Parameters
+
+- Required parameters:
+  - `[<db_name>.]<dict_table>`: The name of the dictionary table, which needs to be a Primary Key table. The supported data type is VARCHAR.
+  - `key_column_expr_list`: The expression list for key columns in the dictionary table, including one or multiple `key_column_exprs`. The `key_column_expr` can be the name of a key column in the dictionary table, or a specific key or key expression.
+
+    This expression list needs to include all Primary Key columns of the dictionary table, which means the total number of expressions needs to match the total number of Primary Key columns in the dictionary table. So when the dictionary table uses Composite Primary Key, the expressions in this list needs to correspond to the Primary Key columns defined in the table schema by sequence. Multiple expressions in this list are separated by commas (`,`). And if a `key_column_expr` is a specific key or key expression, its type must match the type of the corresponding Primary Key column in the dictionary table.
+
+- Optional parameters:
+  - `<value_column>`: The name of the value column, which is also the mapping column. If the value column is not specified, the default value column is the AUTO_INCREMENT column of the dictionary table.  The value column can also be defined as any column in the dictionary table excluding auto-incremented columns and primary keys. The column's data type has no restrictions.
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/d56c8875-3871-4a30-a91b-7382f832642b)
-> the auto-incremented column
Use the same term or wording for the same thing